app/views/europeana/feedback_button/feedback_mailer/post.text.erb

```diff
@@ -1,4 +1,6 @@
 Type: <%= @type %>
 URL: <%= @page %>
+Email: <%= @email %>
+
 Text:
 <%= @text %>
```

Comment:
let's do the concatenation here once the frontend starts sending the email and text separately.

Reply:
I'm not really sure we need concatenation do we? Doesn't the entire email body just end up in the ticket description in the JIRA service desk? (I haven't seen it, so not sure.)

Reply:
sure, adding the email to the text part of the mail was based on the acceptance criteria, but I agree that it's sufficient if not better when formatted with the label as we do now.